content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,37 +1,32 @@
 # Sharing Experiments
 
-There are two types of remotes that can store experiments. Git remotes are
-distributed copies of the Git repository, for example on GitHub or GitLab.
-
-[DVC remotes](/doc/command-reference/remote) on the other hand are
-storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
-configure with `dvc remote`. DVC uses them to store and fetch large files that
-don't normally fit inside Git repos.
-
-DVC needs both kinds of remotes for backing up and sharing experiments.
-
-Experiment files that are normally tracked in Git (like code versions) are
-shared using Git remotes, and files or directories tracked with DVC (like
-datasets) are shared using DVC remotes.
-
-> See [Git remotes guide] and `dvc remote add` for information on setting them
-> up.
+Two types of _remotes_ are needed to upload experiments for sharing.
+[Git remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes)
+are distributed copies of the Git repository, hosted for example on GitHub or
+GitLab. Small files like experimental code and
+[DVC metafiles](/doc/user-guide/project-structure) files will go there.
+[DVC remotes](/doc/command-reference/remote) on the other hand are data storage
+locations (e.g. Amazon S3 or Google Drive). You can use them to back up and
+[share data](/doc/use-cases/sharing-data-and-model-files) files and directories
+that don't fit inside Git repos.
+
+> See this [Git remotes guide] and `dvc remote add` for info. on setting them
+> up.  
+> ⚠️ Note that only [SSH Git URLs] support DVC experiment sharing.
```

Comment:
> we should point out that dvc exp push/dvc exp pull only work with SSH

Good call @daavoo. Noted!

Reply:
p.s. do you know if there's an issue to update this info in the cmd ref? Cc @pmrowla @efiop if someone wouldn't mind doing that pls 😁

Reply:
https://github.com/iterative/dvc.org/pull/2731

Reply:
Thanks @daavoo